consensus/polybft/consensus_runtime.go

```diff
@@ -82,6 +82,8 @@ type runtimeConfig struct {
 	txPool                txPoolInterface
 	bridgeTopic           topic
 	numBlockConfirmations uint64
+	IsRelayer             bool
+	RPCEndpoint           string
```

Comment:
I will just have `consensusConfig       *consensus.Config` inside `type runtimeConfig struct`
No need to have one more structure

Reply:
Nope, If you put it inside `PolyBFTConfig`, where it belongs IMO, you would already have it there, because runtimeConfig contains `PolyBFTConfig` reference. However, your call.

Reply:
we need to move 
```
	// IsRelayer is true if node is relayer
	IsRelayer bool `json:"isRelayer,omitempty"`

	// RPCEndpoint
	RPCEndpoint string `json:"rpcEndpoint,omitempty"`
```	
to `PolybftConfig` and set those at genesis. We can do that in separate PR